### DISCUSSION TITLE
Keyboard and Languages not show Manage Input Methods tab

### DISCUSSION THREAD

**Discussion by jobegod** [linuxdeepin/developer-center#2923] [Features Request & Ideas | 特性请求 & 头脑风暴]
After upgrade deepin 15.11 to 20.2.4 Keyboard and Languages not show Manage Input Methods tab
![image](https://user-images.githubusercontent.com/84620905/140796038-37de200e-9588-4319-af2e-06b264039c17.png)

**Comment by Zeno-sole** [comment]
@jobegod  Do you still use deepin? We are about to release 20.5. This version has many new features, and there are also input and hair tabs.

**Reply by jobegod** [reply]
Yes, I use deepin and upgrade my above system to 20.4 (15.11 --> 20.2.4 --> 20.3 --> 20.4) but I have same issue. When release 20.5 I upgrade it to view changes.

Thanks!
